examples/metatensor-plumed/lj-crystal.py

```diff
@@ -0,0 +1,543 @@
+import ase.optimize
+import ase.units
+
+#
+import chemiscope
+import featomic.torch
+import matplotlib.pyplot as plt
+import metatensor.torch as mts
+import metatomic.torch as mta
+import numpy as np
+import torch
+
+
+# %%
+# Simulation Starting Point: Random Structure w/ Energy Minimized
+# ---------------------------------------------------------------
+#
+# We start with a random cloud of 38 Argon atoms. To get a more reasonable
+# starting point for our MD simulation, we'll perform a quick energy
+# minimization using the FIRE optimizer. This relaxes the structure into a
+# nearby local minimum, mitigating artificial effects from an unphysical
+# randomized structure.
+
+np.random.seed(0xDEADBEEF)
+atoms = ase.Atoms(
+    symbols=["Ar"] * 38,
+    positions=2 * np.random.rand(38, 3),
+    # Set a cell without pbc to silence a warning from PLUMED
+    cell=10 * np.eye(3),
+    pbc=False,
+)
+
+lj_potential = ase.calculators.lj.LennardJones(rc=2.5)
+atoms.calc = lj_potential
+optimizer = ase.optimize.FIRE(atoms)
+optimizer.run(fmax=0.8)
+
+# %%
+# The Target Structures
+# ---------------------
+#
+# The two most famous structures for LJ38 are the global minimum (a perfect
+# truncated octahedron) and a lower-symmetry icosahedral structure which is a
+# deep local minimum. Let's load them and visualize all three (our starting
+# structure and the two targets) using `chemiscope`.
+
+minimal = ase.io.read("lj-oct-0k.xyz")
+# FIXME: this does not look like the "other" stable structure
+other = ase.io.read("lj38.xyz")
+
+settings = {"structure": [{"playbackDelay": 50, "unitCell": True, "bonds": False}]}
+chemiscope.show([minimal, other, atoms], mode="structure", settings=settings)
+
+
+# %%
+# Defining our custom collective variable
+# ---------------------------------------
+#
+# To distinguish between the liquid-like state and the highly ordered
+# face-centered cubic (FCC) packing of the global minimum, we use **Steinhardt
+# order parameters**, specifically :math:`Q_4` and :math:`Q_6`. These parameters
+# are rotationally invariant and measure the local orientational symmetry around
+# each atom.
+#
+# - :math:`Q_6` is often high for both icosahedral and FCC-like structures,
+#   making it a good measure of general "solidness".
+# - :math:`Q_4` helps to distinguish between different crystal packing types. It
+#   is close to zero for icosahedral structures but has a distinct non-zero value
+#   for FCC structures.
+#
+# We will build a model that calculates global, system-averaged versions of these
+# parameters.
+
+
+# %%
+#
+# Encapsulating the Logic in a ``torch.nn.Module``
+# '''''''''''''''''''''''''''''''''''''''''''''''''
+#
+# To make this CV usable by PLUMED via the ``METATOMIC`` interface, we must wrap
+# our calculation logic in a ``torch.nn.Module``. This class takes a list of
+# atomic systems and returns a `metatensor.TensorMap` containing the calculated
+# CV values. The interface is defined in [TODO link].
+
```

Comment:
it could be worth to explain why using `SphericalExpansion` together with `selected_keys` and the final norm gives something similar (but not quite the same) to q4/q6